FFCDemoPaymentService/Scheduling/CreateSchedule.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Collections.Generic;
+using FFCDemoPaymentService.Models;
+using FFCDemoPaymentService.Data;
+using Amqp;
+
+namespace FFCDemoPaymentService.Scheduling
+{
+    class CreateSchedule : IScheduling
```

Comment:
I think this logic should sit in the message service and it can be condensed.

The MessageService receives a message object which contains a Json string that needs to be deserialized to a C# class.  The ticket for receiving from schedule queue will handle that, but if you're doing this story first you probably need to create that class.  I'd call it Claim and have one property ClaimId (mapped from claimId) and ignore anything else in there.

The message service can then pass the claimId to the ScheduleService create claim along with the current date and the number of payments to schedule.

Reply:
not sure i full understand. ill pick up with you later